### DISCUSSION TITLE
Adding FFS module

### DISCUSSION THREAD

**Discussion by Name1t** [xCuri0/ReBarUEFI#49] [Q&A]
Hi, I'm trying to add the FFS module (Gigabyte A320M-S2H V2 motherboard), but in my case the last module is `freeform` subtype, not `DXE driver'. Should I add it to that module? Thanks!

![Screenshot 2023-04-30 101744](https://user-images.githubusercontent.com/54121593/235340973-71fc24be-550c-4405-8506-59b6f5a6430a.png)

**Comment by xCuri0** [comment]
Yes it should work, as long as it's the same volume as PciBus

**Reply by deb4rgho** [reply]
Hi, I am also trying to get ReBar enabled on my A320M-S2H motherboard. I saw some tutorials where it was shown to use a bin file in uefitool but im unable to find a bin file for my motherboard. I am very new to these things and only doing this because i bought an intel gpu without the knowledge of ReBar.
